axelor-redmine/src/main/java/com/axelor/apps/redmine/service/sync/timeentries/RedmineImportTimeSpentServiceImpl.java

```diff
@@ -352,8 +352,14 @@ public void setTimesheetLineFields(TimesheetLine timesheetLine, TimeEntry redmin
     timesheetLine.setDate(redmineSpentOn);
 
     String value = redmineCustomFieldsMap.get(redmineTimeSpentDurationForCustomer);
-    timesheetLine.setDurationForCustomer(
-        value != null && !value.equals("") ? new BigDecimal(value) : duration);
+
+    if (value != null && !value.trim().isEmpty()) {
```

Comment:
in order to avoid code duplication, you can do:

```
value = Optional.ofNullable(value).orElseGet(String::new).trim();
if(!value.isEmpty()){
  ...
}
```